core/runtime/src/main/java/io/quarkiverse/langchain4j/guardrails/OutputGuardrailResult.java

```diff
@@ -0,0 +1,154 @@
+package io.quarkiverse.langchain4j.guardrails;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class OutputGuardrailResult implements GuardrailResult<OutputGuardrailResult> {
+
+    private static final OutputGuardrailResult SUCCESS = new OutputGuardrailResult();
+
+    private final Result result;
+    private List<Failure> failures;
```

Comment:
should be guarded - actually, can it be final?

Reply:
I made it final.